source/common/formatter/substitution_formatter.cc

```diff
@@ -1541,6 +1551,36 @@ ProtobufWkt::Value PlainStringFormatter::formatValue(const Http::RequestHeaderMa
   return str_;
 }
 
+PlainNumberFormatter::PlainNumberFormatter(const double& num) { num_.set_number_value(num); }
```

Comment:
I think just `double num` is fine here. I don't see any benefit to using a reference type.

Reply:
Done.